packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+                    "type": "image/png"
+                }
+            ],
+            "self": {
+                "href": "http://${yourOktaDomain}/api/v1/catalog/apps/facebook"
+            }
+        }
+    },
+    {
+        "name": "24 Seven Office 0",
+        "status": "ACTIVE",
+        "id": "0oasrudLtMlzAsTxk0g3",
+        "_links": {
+            "self": {
+                "href": "http://${yourOktaDomain}/api/v1/apps/0oasrudLtMlzAsTxk0g3"
+            }
+        }
+    }
+]
+```
+
+#### Add an App target to an App administrator role
+
+##### Add an App target to an App administrator role given to a User
+
+<ApiOperation method="put" url="/api/v1/users/${userId}/roles/${roleId}/targets/catalog/apps/${appName}" />
+
+Adds an App target for an `APP_ADMIN` Role assigned to a User
+
+When you add the first App target, you reduce the scope of the role assignment. The Role no longer applies to all App targets, but applies only to the specified target.
+
+Adding an App target overrides any existing App Instance targets of the App. For example, if someone was assigned to administer a specific Facebook instance, calling this endpoint with `facebook` for `appName`, would make them administrator for all Facebook instances.
+
+###### Request parameters
+
+
+| Parameter   | Description                                                  | Param Type   | DataType   | Required |
+| :---------- | :----------------------------------------------------------- | :----------- | :--------- | :------- |
+| appName     | `name` of App target from catalog to scope role assignment   | URL          | String     | TRUE     |
+| roleId      | `id` of a Role                                               | URL          | String     | TRUE     |
+| userId      | `id` of a User                                               | URL          | String     | TRUE     |
+
+###### Response parameters
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+###### Request example
+
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/users/00u6fud33CXDPBXULRNG/roles/KVJUKUS7IFCE2SKO/targets/catalog/apps/amazon_aws"
+```
+
+###### Response example
+
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+##### Add an App target to an App administrator role given to a Group
+
+<ApiOperation method="put" url="/api/v1/groups/${groupId}/roles/${roleId}/targets/catalog/apps/${appName}" />
+
+Adds an App target for an `APP_ADMIN` Role assigned to a Group
+
+When you add the first App target, you reduce the scope of the role assignment. The Role no longer applies to all App targets, but applies only to the specified target.
+
+Adding an App target overrides any existing instance targets of the app. For example, if someone was assigned to administer a specific Facebook instance, calling this endpoint with `facebook` for `appName`, would make them administrator for all Facebook instances.
```

Comment:
was assigned > is assigned
calling this endpoint > a call to this endpoint
remove comma before "would"
would make them administrator > would make that person the administrator